.gitignore

```diff
@@ -1 +1,2 @@
 elm-stuff/
+npm-utils/
```

Comment:
Alternatively, I could do this by checking out in different sub folders and modifying the scripts slightly.